### PR TITLE
Add template code

### PR DESCRIPTION
**Pull request by grahambancroft**
Add initial sass and template code

./run to build

### CODE REVIEW

File: static/sass/_sidebar-nav.scss [hunk 1]
```diff
@@ -0,0 +1,277 @@
+@mixin rounded_corners($radius: 4px 4px 4px 4px) {
+  -webkit-border-radius: $radius;
+  -moz-border-radius: $radius;
+  border-radius: $radius;
+}
+
+$button-width: 10px; 
+$gutter: 20px;
```
Comment (anthonydillon) [inline static/sass/_sidebar-nav.scss]: We have `$grid-gutter-width` which should be used here instead.

File: static/sass/_sidebar-nav.scss [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+          border: 0;
+          color: $color-mid-dark;
+          display: block;
+
+          &:hover, &:focus {
+            text-decoration: none;
+          }
+
+          &:hover, &.active {
+            color: #e95420;
```
Comment (anthonydillon) [inline static/sass/_sidebar-nav.scss]: Can you used to use `$color-brand` please.

File: static/sass/_sidebar-nav.scss [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+          }
+        }
+
+        &:after {
+          background-position: center center;
+          background-repeat: no-repeat;
+          height: 1.5rem;
+        }
+        &.collapsed:after {
+          background-image: url('https://assets.ubuntu.com/v1/04d2075a-chevron_down.svg');
```
Comment (anthonydillon) [inline static/sass/_sidebar-nav.scss]: Please use `$assets-path`

File: static/sass/_sidebar-nav.scss [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+          content: '';
+        }
+        &.expanded:after {
+          background-image: url('https://assets.ubuntu.com/v1/f3f43376-chevron_up.svg');
```
Comment (anthonydillon) [inline static/sass/_sidebar-nav.scss]: Please use `$assets-path`

File: static/sass/_sidebar-nav.scss [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
+    }
+  }
+
+  .note {
+    background-color: #fdf6e7;
+    border: 1px solid #eca918;
+    border-radius: 2px;
+    color: inherit;
+    padding: 20px;
+  }
```
Comment (anthonydillon) [inline static/sass/_sidebar-nav.scss]: Let remove and use Vanilla notifications: https://github.com/ubuntudesign/vanilla-framework/blob/master/scss/_patterns_notifications.scss

File: static/sass/global.scss [hunk 6]
```diff
@@ -1,3 +1,380 @@
...
+body {
+  background: $color-light; // temporary style
+}
+
+.theme__wrap,
+.p-footer__inner {
+  margin: 0 auto;
+  
+  @media only screen and (min-width: $breakpoint-large) {
+    width: 1024px;
```
Comment (anthonydillon) [inline static/sass/global.scss]: Can we use `$breakpoint-large`?

File: static/sass/global.scss [hunk 7]
```diff
@@ -1,3 +1,380 @@
...
+
+.theme__inner {
+  margin: 0 auto;
+}
+
+.theme {
+  .p-navigation {
+    background: $color-brand;
+    border-color: $color-mid-light;
+    border-bottom-color: #cdcdcd;
```
Comment (anthonydillon) [inline static/sass/global.scss]: `$color-mid-light` please

File: static/sass/global.scss [hunk 8]
```diff
@@ -1,3 +1,380 @@
...
+
+    @media only screen and (min-width: $breakpoint-large) {
+      border-bottom: 0;
+      margin-bottom: 30px; // temporary style
+    }
+    
+    .p-navigation__toggle--open,
+    .p-navigation__toggle--close {
+      overflow: hidden;
+      background-image: url("https://assets.ubuntu.com/sites/ubuntu/latest/u/img/icons/navigation-menu-plain.svg");
```
Comment (anthonydillon) [inline static/sass/global.scss]: Use `$assets-path` and use single quotes please

File: static/sass/global.scss [hunk 9]
```diff
@@ -1,3 +1,380 @@
...
+      }
+    }
+    
+    .p-navigation__inner {
+      border-color: $color-mid-light;
+      width: auto;
+        
+      @media only screen and (min-width: $breakpoint-medium) {
+        margin: 0 auto;
+        width: 1024px;
```
Comment (anthonydillon) [inline static/sass/global.scss]: `$breakpoint-medium`?

File: static/sass/global.scss [hunk 10]
```diff
@@ -1,3 +1,380 @@
...
+      }
+    }
+    
+    &__links {
+      background: #f0f0f0;
```
Comment (anthonydillon) [inline static/sass/global.scss]: Can you darken `$color-light` a little?

File: static/sass/global.scss [hunk 11]
```diff
@@ -1,3 +1,380 @@
...
+        position: absolute;
+        width: 1px;
+      }
+    }
+    
+    .p-navigation__link,
+    .p-navigation__link:visited,
+    .p-navigation__link:link { 
+      background: #f0f0f0;
+      border-right: 1px solid #cdcdcd;
```
Comment (anthonydillon) [inline static/sass/global.scss]: `$color-mid-light`

File: static/sass/global.scss [hunk 12]
```diff
@@ -1,3 +1,380 @@
...
+        top: 0;
+        position: absolute;
+        width: 1px;
+      }
+    }
+    
+    .p-navigation__link,
+    .p-navigation__link:visited,
+    .p-navigation__link:link { 
+      background: #f0f0f0;
```
Comment (anthonydillon) [inline static/sass/global.scss]: Can you darken `$color-light` a little?

File: static/sass/global.scss [hunk 13]
```diff
@@ -1,3 +1,380 @@
...
+        font-weight: bold;
+        font-size: .875rem;
+        width: auto;
+        
+        &:last-of-type { 
+          border-right: 1px solid darken($color-brand, 6%);
+        }
+      }
+      
+      overflow: hidden;
```
Comment (anthonydillon) [inline static/sass/global.scss]: Can you move this style up to the main style block.

File: static/sass/global.scss [hunk 14]
```diff
@@ -1,3 +1,380 @@
...
+    }
+  }
+  
+  .p-navigation .p-navigation__logo .p-navigation__link:hover {
+    background: none;
+    text-decoration: none;
+  }
+  
+  .p-navigation .p-navigation__link:hover {
+    background: #f8f8f8;
```
Comment (anthonydillon) [inline static/sass/global.scss]: `$color-light`

File: static/sass/global.scss [hunk 15]
```diff
@@ -1,3 +1,380 @@
...
+    border-color: #cdcdcd;
```
Comment (anthonydillon) [inline static/sass/global.scss]: `$color-mid-light`

File: static/sass/global.scss [hunk 16]
```diff
@@ -1,3 +1,380 @@
...
+  &:visited {
+    border: 1px solid $color-mid-dark;
+    color: inherit;
+  }
+}
+
+// cards
+.row--cards {
+  &__title {
+    background: rgba(0, 0, 0, 0) url("https://assets.ubuntu.com/v1/9334aabe-external-link-cool-grey.svg") no-repeat scroll left center / 16px auto;
```
Comment (anthonydillon) [inline static/sass/global.scss]: Please use `$assets-path` and single quotes.

File: static/sass/global.scss [hunk 17]
```diff
@@ -1,3 +1,380 @@
...
+    }
+
+    &__date {
+      color: $color-mid-dark;
+      margin-top: -1rem;
+    }
+  }
+}
+
+$grid-border-style: 1px dotted $color-mid-light;
```
Comment (anthonydillon) [inline static/sass/global.scss]: Can you move this variable into the `.grid-list` component to scope it to that component please.

File: templates/includes/base/header.html [hunk 18]
```diff
@@ -1,19 +1,33 @@
...
+              developer
+            </a>
         </div>
-    </nav>
+        <a href="#navigation" class="p-navigation__toggle--open" title="menu">Menu</a>
+        <a href="#navigation-closed" class="p-navigation__toggle--close" title="close menu">Menu</a>    <nav id="navigation" class="p-navigation__nav">
```
Comment (anthonydillon) [inline templates/includes/base/header.html:11]: Can you pop the nav on its own line please.

File: templates/pages/examples/summary.html [hunk 19]
```diff
@@ -4,44 +4,75 @@
...
+            <div class="col-12">
+                <h5><a href="#">Server&nbsp;&rsaquo;</a></h5>
+                <p>Pellentesque aliquam risus felis. Etiam porta nibh sit amet urna ornare posuere nec a augue.</p> 
+            </div>               
         </li>
     </ul>
+</div>
 
+<div class="row row--cards u-no-padding--left">
+    <div class="col-8 u-no-margin--left">
```
Comment (anthonydillon) [inline templates/pages/examples/summary.html]: Do we need `col-8` here?

File: templates/pages/examples/summary.html [hunk 20]
```diff
@@ -4,44 +4,75 @@
...
-        <h1>Summary</h1>
-        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque aliquam risus felis. Etiam porta nibh sit amet urna ornare posuere nec a augue. Interdum et malesuada fames ac ante ipsum primis in faucibus.</p>
-    </div>
-
-</div>
-
-
-<div class="row u-no-padding--left">
-
     <div class="col-12">
```
Comment (anthonydillon) [inline templates/pages/examples/summary.html]: Do we need `col-12` here?

File: templates/pages/examples/tutorials.html [hunk 21]
```diff
@@ -4,43 +4,36 @@
...
-        <p>Get an easy onboarding on &lt;AREA&gt; and other areas with our set of tutorials. They are available on our central tutorials website for any level of experience, and we are sure they will whet your appetite!</p>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque aliquam risus felis. Etiam porta nibh sit amet urna ornare posuere nec a augue. Interdum et malesuada fames ac ante ipsum primis in faucibus. Duis dapibus molestie ultricies. Etiam laoreet, dui vel cursus blandit, orci ante mollis ex, et pretium enim justo vitae diam.</p>
     </div>
-
 </div>
 
 <div class="row u-no-padding--left">
-
-    <ul class="u-clearfix u-no-margin--left u-no-padding--left col-12">
+    <ul class="p-list u-clearfix col-12">
```
Comment (anthonydillon) [inline templates/pages/examples/tutorials.html:14]: Do we need this wrapping col?

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Some comments in code. Please ignore if the file is to be removed.

**Comment by WillMoggridge** [comment]
Can I change this to just merge into master? I don't think we need a dev branch

**Review by anthonydillon — APPROVED** [review]

**Comment by anthonydillon** [comment]
Looks good. Can we merge with master as @WillMoggridge asks?

**Pull request merged**